packages/@lwc/rollup-plugin/src/index.js

```diff
@@ -61,6 +60,10 @@ module.exports = function rollupLwcCompiler(pluginOptions = {}) {
             // Note that in @rollup/plugin-node-resolve v13, relative imports will sometimes
             // be in absolute format (e.g. "/path/to/module.js") so we have to check that as well.
             if ((importee.startsWith('.') || importee.startsWith('/')) && importer) {
+                const { scoped, filename } = parseQueryParamsForScopedOption(importee);
+                if (scoped) {
+                    importee = filename; // remove query param
+                }
```

Comment:
I suppose someday scoped styles could be imported from another package (`import css from 'other-package/styles.scoped.css'`), but it's not possible today, so I think it's fine to put the check within this `if` block.